### PR TITLE
Add clair scanning

### LINKED ISSUE #62: Add clair scanning

### PR DESCRIPTION
**Pull request by alexandre-cartapanis**
Closes #62

### CODE REVIEW

File: .github/workflows/java-microservice-ci-default.yml [hunk 1]
```diff
@@ -360,7 +360,35 @@ jobs:
...
+        uses: actions/upload-artifact@v4
+        with:
+          name: clair-docker-db
+          path: ${{github.workspace}}/clair_docker_db.sarif
+
+      - name: Clair scan docker service
+        uses: quay/clair-action@main
+        with:
+          image-ref: 'bruxless/${{ inputs.project_name }}-service:${{ steps.get_version.outputs.project_version }}'
+          output: "clair_docker_service.sarif"
```
Comment (sebastienrospars) [inline .github/workflows/java-microservice-ci-default.yml:382]: Add way to disable it like trivy

File: .github/workflows/java-microservice-ci-default.yml [hunk 2]
```diff
@@ -360,7 +360,35 @@ jobs:
           name: docker-sbom-${{ inputs.project_name }}-service.json
           path: ${{github.workspace}}/target/docker-sbom-${{ inputs.project_name }}-service.json
 
-      # Dependencies checking
+      # Clair checking
+      - name: Clair scan docker db
+        uses: quay/clair-action@main
```
Comment (sebastienrospars) [inline .github/workflows/java-microservice-ci-default.yml:365]: I prefer fix version than main

Comment (alexandre-cartapanis) [reply .github/workflows/java-microservice-ci-default.yml]: Corrected but documentation is using main, it is maybe better while the action is still in early development

File: .github/workflows/java-microservice-ci-default.yml (hunk 1 above)
Comment (sebastienrospars) [reply .github/workflows/java-microservice-ci-default.yml:382]: I got confused with enable_database_docker. It's ok for me

**Review by sebastienrospars — APPROVED** [review]

**Pull request merged**